projects/packages/videopress/src/client/block-editor/blocks/video/components/player/index.native.js

```diff
@@ -104,26 +109,37 @@ export default function Player( { isSelected, attributes } ) {
 		}
 	}, [] );
 
-	const loadingStyle = {};
-	if ( ! isPreviewReady( preview ) ) {
-		loadingStyle.height = 250;
+	const containerLoadingStyle = {};
+	if ( ! isPreviewAvailable( preview ) ) {
+		containerLoadingStyle.height = loadingStyle.height;
+		containerLoadingStyle.backgroundColor = loadingStyle.backgroundColor;
```

Comment:
There is a flicker between the initial loading screen and when the embed html is rendered. I'm not sure how prevalent this is on a production build but using the loading background reduces the flicker.